docs/scripts/d2_svg_file.py

```diff
@@ -21,7 +22,7 @@ def on_config(config: MkDocsConfig) -> MkDocsConfig:
     original_render = plugin.renderer
     plugin.keys = set()
 
-    def new_render(source, opts, alt):  # noqa: ANN001, ANN202
+    def new_render(source, opts, alt):  # noqa: ANN001, ANN202 # pyright:ignore[reportUnknownParameterType,reportMissingParameterType]
```

Comment:
Here and in `docs/scripts/gen_additional_readmes.py`, why did things only start failing in this PR?

Reply:
The validation is only in my editor, I just wanted to resolve the squiggles. This validation was off because imo we don't need strict type checking for one-off scripts like this.